h2o-core/src/main/java/water/init/NetworkInit.java

```diff
@@ -30,6 +30,7 @@
  */
 public class NetworkInit {
 
+  public static String DISABLE_NON_LEADER_API = H2O.OptArgs.SYSTEM_PROP_PREFIX + "disableNonLeaderApi";
```

Comment:
This is a system-level property, not directly user facing. One can always do -Dsys.ai.h2o.disableapinonleader=true, but users should not care about it. Therefore not taking it into H2o.args.